Incompatibility with vue 2.7

After upgrading to `vue:2.7`, [which has composition-api built in](https://blog.vuejs.org/posts/vue-2-7-naruto.html), I started getting this compilation error when running my app:

```
 ERROR  Failed to compile with 1 error                                                                                                                                 12:19:59 PM

 error  in ./node_modules/vue-concurrency/dist/vue2/vue-concurrency.module.js

Module not found: Error: Can't resolve '@vue/composition-api' in '/home/micael/code/webapp/node_modules/vue-concurrency/dist/vue2'

ERROR in ./node_modules/vue-concurrency/dist/vue2/vue-concurrency.module.js 1:0-140
Module not found: Error: Can't resolve '@vue/composition-api' in '/home/micael/code/webapp/node_modules/vue-concurrency/dist/vue2'
 @ ./node_modules/vue-concurrency/dist/index.module.js 1:0-45 1:0-45
 @ ./node_modules/babel-loader/lib/index.js!./node_modules/ts-loader/index.js??clonedRuleSet-41.use[1]!./node_modules/@vue/vue-loader-v15/lib/index.js??vue-loader-options!./src/components/form/CFileUpload.vue?vue&type=script&lang=ts& 5:0-42 9:26-33
 @ ./src/components/form/CFileUpload.vue?vue&type=script&lang=ts& 1:0-252 1:268-271 1:273-522 1:273-522
 @ ./src/components/form/CFileUpload.vue 2:0-63 3:0-58 3:0-58 9:2-8
 @ ./src/components/ sync [A-Z]\w+\.(vue%7Cjs)$ ./form/CFileUpload.vue
 @ ./src/main.ts 99:21-81
```

Is there anything I'm missing to make this work? Or is `vue-concurrency` incompatible with `vue:2.7`?

--- a/README.md
+++ b/README.md
@@ -12,8 +12,8 @@ vue-concurrency aims to provide a reasonable abstraction for performing asynchro
 
 ## Features
 
-- Vue 3 ✅
-- Vue 2 + [@vue/composition-api](https://github.com/vuejs/composition-api) ✅
+- Vue 3 + Vue 2.7 (Version >= 4.x)✅
+- Vue 2 + [@vue/composition-api](https://github.com/vuejs/composition-api) (Version < 4.x) ✅
 - **TypeScript support**
 - **Async cancellation** via generator functions and [CAF](https://github.com/getify/CAF)
 - Providing `AbortSignal` to **abort XHR/Fetch requests**

